### PR TITLE
Set volume api

### PR DESCRIPTION
**Pull request by skywojciechowskim**
Summary: Implementation of get/setVolume in IMediaPipeline.
Type: Feature
Owner: Marcin Wojciechowski
Reviewers: Adam Czynszak, Luke Williamson
Coding Standard Applied: Y
Test Plan: None
Dependencies and Impacts: None
Jira: RIALTO-6

### CODE REVIEW

File: media/server/gstplayer/public/IGstPlayer.h [hunk 1]
```diff
@@ -199,6 +199,23 @@ class IGstPlayer
      *
      */
     virtual void renderFrame() = 0;
+
+    /**
+     * @brief Set level and transition of audio attenuation.
+     *        Sets the current volume for the pipeline (0.0 silent -> 1.0 full volume)
+     *
+     * @param[in] volume Target volume level (0.0 - 1.0)
```
Comment (skywilliamsonl) [inline media/server/gstplayer/public/IGstPlayer.h:207]: Add ':' in param description here and in getVolume below.
Add retval.

File: tests/media/client/ipc/mediaPipelineIpc/GetVolumeTest.cpp [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ */
+TEST_F(RialtoClientMediaPipelineIpcGetVolumeTest, Success)
+{
+    expectIpcApiCallSuccess();
+
+    EXPECT_CALL(*m_channelMock, CallMethod(methodMatcher("getVolume"), m_controllerMock.get(),
+                                           GetVolumeRequestMatcher(m_sessionId), _, m_blockingClosureMock.get()));
+
+    double volume;
+    EXPECT_TRUE(m_mediaPipelineIpc->getVolume(volume));
```
Comment (skywilliamsonl) [inline tests/media/client/ipc/mediaPipelineIpc/GetVolumeTest.cpp:57]: Set response and check volume param set correctly.

File: media/server/gstplayer/public/IGstPlayer.h (hunk 1 above)
Comment (skywojciechowskim) [reply media/server/gstplayer/public/IGstPlayer.h]: done

File: tests/media/client/ipc/mediaPipelineIpc/GetVolumeTest.cpp (hunk 2 above)
Comment (skywojciechowskim) [reply tests/media/client/ipc/mediaPipelineIpc/GetVolumeTest.cpp]: done

**Review by skywilliamsonl — APPROVED** [review]

**Review by aczs — APPROVED** [review]

**Pull request merged**